common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -361,6 +391,84 @@ def test_can_reset_attempts(self):
         exam_attempts_section.remove_student_attempt()
         self.assertFalse(exam_attempts_section.is_student_attempt_visible)
 
+    @attr('a11y')
+    def test_track_selection_a11y(self):
+        """
+        Proctored: Track selection accessibility tests
+        """
+        self.track_selection_page.a11y_audit.config.set_rules({
```

Comment:
You do bring up a good point in that I should limit the scope of the checks to the tab content area so that it's not repeating the full page check each time. Let me investigate.

I do see several `visit()`'s though, one for each section (in addition to the tab switch or section activation).